instance/github.py

```diff
@@ -157,8 +164,39 @@ def __init__(self, number, fork_name, branch_name, title, username, body=''):
         self.body = body
 
     @property
+    def settings(self):
+        """
+        Settings contained in the PR body
+        """
+        return yaml.load(get_settings_from_pr_body(self.body)) or {}
+
+    @property
+    def database_url(self):
+        """
+        An optional database url, from the DATABASE_URL setting in the PR body
+        """
+        return self.settings.get(self.SETTINGS['DATABASE_URL'])
```

Comment:
I'm not entirely sure I like conflating the Ansible settings with other non-Ansible settings.  It might be better to keep them separate.  (Namespace conflation always starts with small steps like this one, and one day you end up with GNU make. :) )